docs/instrumentations/php/native.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: 'Automatic Instrumentation'
+sidebarTitle: 'Native'
+icon: 'php'
+---
+
+<Info>
+  Native Instrumentation is currently the default setting of Odigos **Open-Source** version.
+
+  eBPF-Based Instrumentation is currently not supported for this language.
+</Info>
+
+## Supported Versions
+
+The minimal supported version of PHP is **8.0.0**.
```

Comment:
same as above, mention latest version we support, and we can update the docs once 8.6 or any other future version is pulled into our php agent.